lib/cretonne/src/verifier/mod.rs

```diff
@@ -409,6 +411,33 @@ impl<'a> Verifier<'a> {
                             got);
             }
         }
+        // We also verify if the postorder defined by `DominatorTree` is sane
+        for (index, (&true_ebb, &test_ebb)) in
+            self.domtree
+                .cfg_postorder()
+                .iter()
+                .zip(domtree.cfg_postorder().iter())
+                .enumerate() {
+            if true_ebb != test_ebb {
```

Comment:
Unfortunately, this won't detect if one PO is a prefix if the other. The `zip` iterator stops when either input ends.

Note that slices implement `Eq`, so you can just compare them with `==`.